Index: app/src/main/java/com/example/lifesum/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.lifesum\r\n\r\nimport androidx.appcompat.app.AppCompatActivity\r\nimport android.os.Bundle\r\nimport kotlinx.android.synthetic.main.activity_main.*\r\n\r\nclass MainActivity : AppCompatActivity() {\r\n    override fun onCreate(savedInstanceState: Bundle?) {\r\n        super.onCreate(savedInstanceState)\r\n        setContentView(R.layout.activity_main)\r\n    }\r\n}\r\n//Prachi Pardeshi
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/lifesum/MainActivity.kt b/app/src/main/java/com/example/lifesum/MainActivity.kt
--- a/app/src/main/java/com/example/lifesum/MainActivity.kt	(revision c6b40a4a3ace1b21fbf149255c18af48246ec9b5)
+++ b/app/src/main/java/com/example/lifesum/MainActivity.kt	(date 1632931722589)
@@ -10,4 +10,3 @@
         setContentView(R.layout.activity_main)
     }
 }
-//Prachi Pardeshi
\ No newline at end of file
